src/components/header/style.css

```diff
@@ -1,49 +0,0 @@
-.header {
-	padding: 40px 0;
-	min-height: 695px;
```

Comment:
strange min-height of 695px. why not 700px? Please read regarding grids in html and page columns design